messaging/src/androidTest/java/io/lantern/messaging/MessagingTest.kt

```diff
@@ -832,8 +833,12 @@ class MessagingTest : BaseMessagingTest() {
                     )
                     val recvMsg = dog.waitFor<Model.StoredMessage>(
                         sentMsg.dbPath,
-                        "dog should receive message"
-                    )
+                        "dog should receive message with 2 completed attachments"
+                    ) {
+                        it.attachmentsMap.values.count {
+                            it.status == Model.StoredAttachment.Status.DONE
```

Comment:
Because lazy attachments are now encrypted asynchronously, we need to wait for the status to get to DONE